test/Project.toml

```diff
@@ -0,0 +1,4 @@
+[deps]
+AdvancedMH = "5b7e9947-ddc0-4b3f-9b55-0d8042f74170"
+MCMCChains = "c7f686f2-ff18-58e9-bc7b-31028e88f75d"
+StructArrays = "09ab397b-f2b6-538f-b94a-2f83cf4a842a"
```

Comment:
`test/Project.toml` is only supported on Julia >= 1.3 but not on earlier Julia versions:
```suggestion
```

Reply:
I see, I seem to have accidentally committed this